### DISCUSSION TITLE
What class is generating BuildConfig.java? (module :core:network)

### DISCUSSION THREAD

**Discussion by ghost** [android/nowinandroid#1476] [Q&A]
ok so, i want to know because i want to implement buildconfig in my custom module of the app im working for.
Another question would be: how to implement buildconfig in another module other than the main app?

**Comment by JackEblan** [comment]
Just add this
https://github.com/android/nowinandroid/blob/211654ff16aea732cab8fd6148998c2a7293612f/benchmarks/build.gradle.kts#L34

to whatever module you are in.
